R/echart.R

```diff
@@ -63,16 +63,57 @@ echart.data.frame = function(
     x = x, y = y
   ), TOJSON_ARGS = list(pretty = TRUE))
 
+  params$tooltip = list(
```

Comment:
I'd prefer a separate function to do these configurations, e.g. something like `eChart() %>% eConfig(calculable = TRUE) %>% eToolbox(show = TRUE, feature = ...)`. It is fine to hard-code some options inside `echart()`, but we should make it possible for users to change these options at the same time.